external/include/protocols/libcoap/config.h

```diff
@@ -37,11 +37,14 @@
 #undef NDEBUG
 #endif
 
-#ifdef CONFIG_NET_SECURITY_TLS
-#define WITH_MBEDTLS
-#else
-#undef WITH_MBEDTLS
-#endif
+/* coap refers easy_tls not supported anymore
+ * So WITH_MBEDTLS is disabled until change easy_tls
+ */
+/* #ifdef CONFIG_NET_SECURITY_TLS */
```

Comment:
Should we apply another config instead of CONFIG_NET_SECURITY_TLS, to enable mbedtls?

Reply:
it use easy tls which is not supported anymore. so disabling CONFIG_NET_SECURITY_TLS is not solution